### DISCUSSION TITLE
Moved Institutions

### DISCUSSION THREAD

**Discussion by humnet** [community/community#51884] [GitHub Education]
How do I change the institution in my GitHub organization account. I can't add a new email either.

**Comment by samus-aran** [comment]
Hi @humnet 👋 

Can you re-apply using your current institution and create a new organisation. Benefits are not associated to organisations but to the individual .

**Reply by humnet** [reply]
Thanks a lot for your help! We have an organization account and moved from MIT to Berkeley, need to edit institution and email in the account and it's not possible
